### PR TITLE
New connector : Sociabble

### PR DESCRIPTION
**Pull request by mdujardibs**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [X ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Sociabble/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1073 @@
...
+                      "mediaIds": {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        },
+                        "description": "mediaIds"
+                      }
+                    }
+                  },
+                  "description": "",
```
Comment (sriyen-msft) [inline certified-connectors/Sociabble/apiDefinition.swagger.json:133]: @mdujardibs Please provide valid description

Documentation -[Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Sociabble/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1073 @@
...
+                  "title": "Content"
+                },
+                "channelIds": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "The channels that the content is published to",
+                    "title": "Channels"
+                  },
+                  "description": "channelIds"
```
Comment (sriyen-msft) [inline certified-connectors/Sociabble/apiDefinition.swagger.json:143]: @mdujardibs Can it be `Channel Ids` ?

File: certified-connectors/Sociabble/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1073 @@
...
+                        "x-ms-summary": "Is default language"
+                      },
+                      "mediaIds": {
+                        "type": "array",
+                        "items": {
+                          "type": "string",
+                          "description": "The unique identifier from a previous media upload to use"
+                        },
+                        "title": "image",
+                        "description": ""
```
Comment (sriyen-msft) [inline certified-connectors/Sociabble/apiDefinition.swagger.json:214]: @mdujardibs Please provide valid description value.

Documentation -[Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Sociabble/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,1073 @@
...
+                      }
+                    },
+                    "required": [
+                      "description",
+                      "language",
+                      "summary",
+                      "title"
+                    ],
+                    "description": "",
```
Comment (sriyen-msft) [inline certified-connectors/Sociabble/apiDefinition.swagger.json:223]: @mdujardibs Please provide valid description value.

Documentation -[Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Sociabble/apiDefinition.swagger.json (hunk 1 above)
Comment (mdujardibs) [reply certified-connectors/Sociabble/apiDefinition.swagger.json:133]: Modifications done.

File: certified-connectors/Sociabble/apiDefinition.swagger.json (hunk 3 above)
Comment (mdujardibs) [reply certified-connectors/Sociabble/apiDefinition.swagger.json]: Done

File: certified-connectors/Sociabble/apiDefinition.swagger.json (hunk 4 above)
Comment (mdujardibs) [reply certified-connectors/Sociabble/apiDefinition.swagger.json]: Done

File: certified-connectors/Sociabble/apiDefinition.swagger.json (hunk 2 above)
Comment (mdujardibs) [reply certified-connectors/Sociabble/apiDefinition.swagger.json:143]: Done

File: certified-connectors/Sociabble/apiDefinition.swagger.json (hunk 1 above)
Comment (sriyen-msft) [reply certified-connectors/Sociabble/apiDefinition.swagger.json:133]: @mdujardibs Can you please push your changes to the PR, the description values are not fixed

File: certified-connectors/Sociabble/apiDefinition.swagger.json (hunk 2 above)
Comment (sriyen-msft) [reply certified-connectors/Sociabble/apiDefinition.swagger.json:143]: @mdujardibs Can you please push your changes to the PR, the description values are not fixed

File: certified-connectors/Sociabble/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,1073 @@
...
+                        "title": "image"
+                      }
+                    },
+                    "required": [
+                      "description",
+                      "language",
+                      "title"
+                    ]
+                  },
+                  "description": "",
```
Comment (sriyen-msft) [inline certified-connectors/Sociabble/apiDefinition.swagger.json:419]: @mdujardibs Can you please fix all the description values ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Sociabble/apiDefinition.swagger.json (hunk 1 above)
Comment (mdujardibs) [reply certified-connectors/Sociabble/apiDefinition.swagger.json:133]: @sriyen-msft I actually did push a commit the 16th of August, https://github.com/microsoft/PowerPlatformConnectors/pull/1026/commits/646008f202b088aaf9b544b9b61602bae34c28c7

Sorry, I am not use  to github, do I need to do something else ?

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**